elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/QueryDsl.scala

```diff
@@ -35,7 +35,7 @@ trait QueryDsl extends DslCommons with SortDsl {
   case class QueryRoot(query: Query,
                        fromOpt: Option[Int],
                        sizeOpt: Option[Int],
-                       sort: Seq[Sort],
+                       sortOpt: Option[Seq[Sort]],
```

Comment:
Ah I see -- I forgot that we had that wrapper layer for backwards compat. Unfortunately, while making this change the first time was backwards compatible, your change isn't backwards compatible (as demonstrated by the fact you had to change the tests). I hope to rework this in 2.0.

To push another change, I think you just need to push it to SplashAttacks:master and it should show up here.